adagrams/game.py

```diff
@@ -1,11 +1,135 @@
+import random
+
 def draw_letters():
-    pass
+ 
+    NUM_TILES = 10
+    letters_grabbed = []
+    letters_remaining_pool = LIST_OF_LETTERS.copy()
+    
+    for i in range(NUM_TILES):
+        # update the last index of the remaining pool list that changes everytime
+        # random.randint stop is inclusive, so -1 which reflect the size of the pool
+        index = len(letters_remaining_pool)-1
+        # pop() allows to return the letter that was taken out and update the remaining pool list
+        letter_grabbed = letters_remaining_pool.pop(random.randint(0,index))
+        letters_grabbed.append(letter_grabbed)
+
+    return(letters_grabbed)
 
 def uses_available_letters(word, letter_bank):
-    pass
+
+    word = word.upper()
```

Comment:
Prefer to call `upper` directly in the for loop instead of adding a line of code to reassign the value of `word` here. 

Like:
```python
for letter in word.upper():
    # rest of your code
```